mitodl/ol-infrastructure
# adding email notification backend

### Description (What does it do?)
Sets the "MITOL_NOTIFICATION_EMAIL_BACKEND" env variable so we can use mailgun

### How can this be tested?
when i send emails directly using
```
from profiles.utils import send_email
send_email(['ambady@mit.edu'], 'test email', 'test', False)
```
it seems to work regardless of this setting. I noticed however, subscription emails would not send. to do that I have a script which manually sets the created date for new learning resources to today and sends subscription updates (which i have not done on prod). You should be able to test this by unsetting this variable (on rc heroku) and then running the following and see that subscription emails dont get sent unless this setting is in place

```
from learning_resources_search.tasks import send_subscription_emails
send_subscription_emails("channel_subscription_type")
```
